scripts/setup-gcp.sh

```
@@ -0,0 +1,71 @@
+read -p "Enter your GCP Project ID: " PROJECT_ID
+
+# Prompt for GCP region
+read -p "Enter your GCP Region (e.g., us-central1): " REGION
+
+# Ensure user is logged in
+echo "Ensuring you're logged into GCP..."
+gcloud auth login
+
+# Set the current project
+echo "Setting project to $PROJECT_ID..."
+gcloud config set project $PROJECT_ID
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)